Application/EdFi.Ods.Features/Composites/CompositeResourceModelBuilder.cs

```diff
@@ -85,7 +85,7 @@ public class CompositeResourceModelBuilder : ICompositeItemBuilder<CompositeReso
                     new ResourceProperty(
                         builderContext.CurrentResource,
                         processorContext.CurrentResourceClass.Name.ToCamelCase() + "Type",
-                        new PropertyType(DbType.AnsiString, 128, 0, 0, true),
+                        new PropertyType(DbType.AnsiString, 128, 0, 0, true, 0),  
```

Comment:
With the default value and placement of the new argument in the `PropertyType` signature, in place for the constructor signature, revert all occurrences of this type of change everywhere (see the large number of differences shown below -- they're all unnecessary).

Reply:
I updated it . Thanks